appveyor.yml

```diff
@@ -5,6 +5,9 @@ version: 1.0.{build}-appveyor
 
 image: Visual Studio 2017
 
+before_build:
```

Comment:
before_build is what codecov recommends.  I don't really have a preference as long as its installed before its used.